todos/src/components/Todos.js

```diff
@@ -1,35 +1,47 @@
-import React from "react";
+import { React } from "react";
 import AddTask from "components/AddTask";
 import Task from "components/Task";
 import "styles/todos.css";
 import { useSelector } from "react-redux";
 import EmptyTaskList from "components/EmptyTaskList";
+import LoadMoreBtn from "components/LoadMoreBtn";
 
 function Todos() {
   const list = useSelector((state) => state.todoReducers.list);
   const isAddTaskVisible = useSelector(
     (state) => state.toggleReducers.isAddTaskVisible
   );
+  const currentPage = useSelector(
+    (state) => state.currentPageReducer.currentPage
+  );
+  const displayedTodoList = list.slice(0, 4 * currentPage);
 
   return (
     <div>
       <div className="all-todos">
-        {isAddTaskVisible && <AddTask/>}
-        {list.map((elem) => {
-          return elem.data ? (
-            <Task
-              key={elem.id}
-              id={elem.id}
-              title={elem.data}
-              isCompleted={elem.isCompleted}
-              date={elem.date}
-              completedDate={elem.completedDate}
-              onEdit={elem.onEdit}
-            />
-          ) : null;
+        {isAddTaskVisible && <AddTask />}
+        {displayedTodoList.map((elem) => {
+          return (
+            elem.data && (
```

Comment:
Why is the `elem.data &&` check is needed here? Is it ever possible that `elem.data` isn't available in the `displayedTodoList`?

Reply:
Yes, Bhai. It is impossible not to have `elem.data` inside the list `displayedTodoList`. Whether there is a list there will be a `key` named `data.` 
Actually, I wrote that before I wrote this line`{list.length === 0 && !isAddTaskVisible && <EmptyTaskList />}` for the logic of showing `EmptyTaskList` component, which was wrong and I accidentally forgot to recheck and remove that. Sorry about that. 
Removing this logic now.